drivers/bch/Kconfig

```diff
@@ -3,16 +3,18 @@
 # see the file kconfig-language.txt in the NuttX tools repository.
 #
 
-menuconfig BCH
-	bool "Block-to-Character (BCH) Support"
-	default n
-	---help---
-		Contains logic that may be used to convert a block driver into
-		a character driver.  This is the complementary conversion as that
-		performed by loop.c.  See include/nuttx/drivers/drivers.h for
-		registration information.
+# Block-to-Character (BCH) Support
+#
+# BCH contains logic that may be used to convert a block driver into
+# a character driver.  This is the complementary conversion as that
+# performed by loop.c.  See include/nuttx/drivers/drivers.h for
+# registration information.
+#
+# The BCH support itself is enabled unless CONFIG_DISABLE_MOUNTPOINT
```

Comment:
How to enable mountpoint without BCH? BCH just waste the flash if user never open the block device directly, but access the block device indirectly through file system.

Reply:
ok. it makes sense. let me think about it a bit.